src/components/Packages/PackagePanelStateText.js

```diff
@@ -52,6 +53,20 @@ class PackagePanelStateText extends Component {
         });
     }
 
+    shouldShowIcon() {
```

Comment:
Please give this a more semantic name (something like `isInteractive`).

Reply:
I think this name is actually very descriptive.  We are not showing an icon at all if the user does not own this package...it doesn't make it not interactive, just hides the icon.